src/main/java/com/laytonsmith/core/functions/World.java

```diff
@@ -66,6 +67,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+@SuppressWarnings("ALL")
```

Comment:
I happened to be looking at this, and suppressing all warnings is generally a bad idea for future code quality.

Reply:
Oh.. I have no clue how this got added. It must have been me trying to suppress a single type of warning, but IntelliJ is StupidJ.